Lib/python/pyinit.swg

```diff
@@ -200,12 +203,29 @@ swig_varlink_type(void) {
       return NULL;
   }
   return &varlink_type;
+#else
+  PyType_Slot slots[] = {
```

Comment:
I get the following error with GCC 7 and 8 from this code:
```
In function ‘PyTypeObject* swig_varlink_type()’:
error: structured binding declaration cannot have type ‘PyType_Slot’
     PyType_Slot slots[] = {
                      ^~
note: type must be cv-qualified ‘auto’ or reference to cv-qualified ‘auto’
error: empty structured binding declaration
error: expected unqualified-id before ‘=’ token
     spec.slots = slots;
                ^
error: expected primary-expression before ‘;’ token
     spec.slots = slots;
                       ^
```
Not sure what is actually causing the error, as it only complains about this specific list initialization of PyType_Slot and not the others. Any idea on how to fix this?